### DISCUSSION TITLE
useFetch not communicating to Docker server

### DISCUSSION THREAD

**Discussion by lucassimines** [nuxt/nuxt#18088] [Questions]
How can I use `useFetch` to fetch my docker server? Using `axios` it works fine, but using `useFetch` it doesn't return anything.

**Comment by danielroe** [comment]
You should provide the full path - e.g. `useFetch('http://localhost:3000/my/docker/api')`.

**Reply by lucassimines** [reply]
> You should provide the full path - e.g. `useFetch('http://localhost:3000/my/docker/api')`.

I provided the full path and it only returns `null` , my `server` api it not being called
